pkg/api/steve/disallow/disallow.go

```diff
@@ -14,7 +14,8 @@ var (
 	// AllowAll is a set of resources for which Rancher doesn't require authentication to perform any operation.
 	AllowAll = map[string]bool{
 		"podsecurityadmissionconfigurationtemplates": true,
-		"projects": true,
+		"projects":                    true,
```

Comment:
Super minor nit, this isn't in alphabetical order

Reply:
Depends on the sorting direction! :)
I wanted projects to be first, since other things are related to it.

Reply:
Makes sense. As long as it's intentional :)